### ISSUE TITLE
An addon to bring back the "mode" heading tab in the editor from April Fools 22

### ISSUE THREAD

**Issue by penguinmoose** [ScratchAddons/ScratchAddons#5129]
### Why this would be helpful/solve a problem

Originally discussed in #5123

It would be nice to bring back the "mode" heading tab from April Fools 22. This includes Cat Blocks (Caturday mode) in addition to Old timey mode, Prehistoric mode, etc. It can be turned off by simply selecting the normal mode.

### What this addon does

This addon brings back the "mode" heading tab from April Fools 22.

### Demonstration(s) and additional context

![Mode](https://en.scratch-wiki.info/w/images/Mode.png)

**Comment by BroJac5246** [comment]
While digging around I found #4463, which brings up the point that we need to make sure that Caturday and Cat Blocks don't conflict.

**Comment by penguinmoose** [comment]
> While digging around I found #4463, which brings up the point that we need to make sure that Caturday and Cat Blocks don't conflict.

An alternative might be when Caturday is enabled **and** cat blocks are enabled as well, all Caturday does is change the Scratch logo.

**Comment by BroJac5246** [comment]
> > While digging around I found #4463, which brings up the point that we need to make sure that Caturday and Cat Blocks don't conflict.
> 
> An solution might be when Caturday is enabled **and** cat blocks are enabled as well, all Caturday does is change the Scratch logo.

It would probably be more efficient to have Cat Blocks disable itself when Caturday is active.

**Comment by mxmou** [comment]
I don't think this addon needs a Caturday option. Instead, the cat blocks addon could change the Scratch logo to the Caturday one.

**Comment by CST1229** [comment]
> I don't think this addon needs a Caturday option. Instead, the cat blocks addon could change the Scratch logo to the Caturday one.

I think the mode button addon should keep Caturday mode, and when combined with cat blocks should enable cat blocks in all modes and remove normal mode.

**Comment by penguinmoose** [comment]
> > I don't think this addon needs a Caturday option. Instead, the cat blocks addon could change the Scratch logo to the Caturday one.
> 
> As an option.

I think the option should be under the Cat Blocks addon - something like "change Scratch logo" If enabled, this will make Cat Blocks change the Scratch logo and remove Caturday mode. I think Cat Blocks should persist in all modes, so Caturday Mode is not almost the same as Cat Blocks. If this option is not enabled, only Caturday Mode will have Cat Blocks.

**Comment by memesong** [comment]
> I think the option should be under the Cat Blocks addon - something like "change Scratch logo" If enabled, this will make Cat Blocks change the Scratch logo and remove Caturday mode. I think Cat Blocks should persist in all modes, so Caturday Mode is not almost the same as Cat Blocks. If this option is not enabled, only Caturday Mode will have Cat Blocks.
> 
> @mxmou I think the problem is that Cat Blocks is too similar to Caturday mode. Maybe there can be a note in Cat Blocks saying that enabling the mode button addon is recommended, rather than Cat Blocks - if that's true.

I think it would be better if you just fused Cat Blocks and Caturday Mode together.

**Comment by BroJac5246** [comment]
> > I think the option should be under the Cat Blocks addon - something like "change Scratch logo" If enabled, this will make Cat Blocks change the Scratch logo and remove Caturday mode. I think Cat Blocks should persist in all modes, so Caturday Mode is not almost the same as Cat Blocks. If this option is not enabled, only Caturday Mode will have Cat Blocks.
> > 
> > @mxmou I think the problem is that Cat Blocks is too similar to Caturday mode. Maybe there can be a note in Cat Blocks saying that enabling the mode button addon is recommended, rather than Cat Blocks - if that's true.
> 
> I think it would be better if you just fused Cat Blocks and Caturday Mode together.

The issue is that Caturday changes the Scratch logo, which not everyone will want.

**Comment by memesong** [comment]
> > > I think the option should be under the Cat Blocks addon - something like "change Scratch logo" If enabled, this will make Cat Blocks change the Scratch logo and remove Caturday mode. I think Cat Blocks should persist in all modes, so Caturday Mode is not almost the same as Cat Blocks. If this option is not enabled, only Caturday Mode will have Cat Blocks.
> > > @mxmou I think the problem is that Cat Blocks is too similar to Caturday mode. Maybe there can be a note in Cat Blocks saying that enabling the mode button addon is recommended, rather than Cat Blocks - if that's true.
> > 
> > 
> > I think it would be better if you just fused Cat Blocks and Caturday Mode together.
> 
> The issue is that Caturday changes the Scratch logo, which not everyone will want.

See, the thing is that there would be an option in the Cat Blocks/Caturday menu to turn on the custom Scratch logo, and if you don't want it, then you could just turn it off.

**Comment by BroJac5246** [comment]
> > [...]
> > The issue is that Caturday changes the Scratch logo, which not everyone will want.
> 
> See, the thing is that there would be an option in the Cat Blocks/Caturday menu to turn on the custom Scratch logo, and if you don't want it, then you could just turn it off.

That would work... I just feel like it would be somewhat unclear.

**Comment by Secret-chest** [comment]
The best way to solve compat is if cat blocks are enabled, to add cat blocks to all modes and let Caturday only change the logo. Or, auto-disable cat blocks when the mode is different from default.
